internal/host/hostcommands/install_cmd_test.go

```diff
@@ -439,7 +439,91 @@ var _ = Describe("installcmd arguments", func() {
 			Expect(stepReply).NotTo(BeNil())
 			verifyArgInCommand(stepReply[0].Args[1], "--installer-args", fmt.Sprintf("'%s'", host.InstallerArgs), 1)
 		})
+	})
+})
 
+var _ = Describe("construct host install arguments", func() {
+	var (
+		cluster *common.Cluster
+		host    *models.Host
+	)
+	BeforeEach(func() {
+		cluster = &common.Cluster{
+			Cluster: models.Cluster{
+				ImageInfo: &models.ImageInfo{},
+			},
+		}
+		host = &models.Host{}
+	})
```

Comment:
Could you also add a test for the `StaticNetworkConfig` case? To make sure that `--copy-network` is added and one where `--copy-network` is already present.

Reply:
These have been already covered at a slightly higher level https://github.com/openshift/assisted-service/blob/11e32547a904a2ba49562b4b0ba3a63e09d50d81/internal/host/hostcommands/install_cmd_test.go#L417. Or did you mean something else?